src/core/core.scale.js

```diff
@@ -1047,12 +1004,149 @@ module.exports = Element.extend({
 				x1 = x2 = borderValue;
 			}
 
-			context.lineWidth = axisWidth;
-			context.strokeStyle = valueAtIndexOrDefault(gridLines.color, 0);
-			context.beginPath();
-			context.moveTo(x1, y1);
-			context.lineTo(x2, y2);
-			context.stroke();
+			ctx.lineWidth = axisWidth;
+			ctx.strokeStyle = valueAtIndexOrDefault(gridLines.color, 0);
+			ctx.beginPath();
+			ctx.moveTo(x1, y1);
+			ctx.lineTo(x2, y2);
+			ctx.stroke();
+		}
+	},
+
+	/**
+	 * @private
+	 */
+	_drawLabels: function(chartArea) {
+		var me = this;
+		var ctx = me.ctx;
+		var optionTicks = me.options.ticks;
+
+		if (!optionTicks.display) {
+			return;
+		}
+
+		var items = me._itemsToDraw || (me._itemsToDraw = me._computeItemsToDraw(chartArea));
+		var tickFont;
+
+		helpers.each(items, function(item) {
+			tickFont = item.font;
+
+			// Make sure we draw text in the correct color and font
+			ctx.save();
+			ctx.translate(item.labelX, item.labelY);
+			ctx.rotate(item.rotation);
+			ctx.font = tickFont.string;
+			ctx.fillStyle = tickFont.color;
+			ctx.textBaseline = 'middle';
+			ctx.textAlign = item.textAlign;
+
+			var label = item.label;
+			var y = item.textOffset;
+			if (helpers.isArray(label)) {
+				for (var i = 0; i < label.length; ++i) {
+					// We just make sure the multiline element is a string here..
+					ctx.fillText('' + label[i], 0, y);
+					y += tickFont.lineHeight;
+				}
+			} else {
+				ctx.fillText(label, 0, y);
+			}
+			ctx.restore();
+		});
+	},
+
+	/**
+	 * @private
+	 */
+	_drawTitle: function(/* chartArea */) {
```

Comment:
I'm wondering why `chartArea` is here commented out. remove it?

Reply:
I agree, it also makes disabling the whole method more complicated:

```javascript
/*
_drawTitle: function(/* chartArea */) {
    var me = this;
    var ctx = me.ctx;
    // ...
}
*/
```

Reply:
Removed